### PR TITLE
[script][tarantula] Adding check if exists at startup, and adding common prefixes

### PR DESCRIPTION
**Pull request by MahtraDR**
I recently got a character that doesn't have a tarantula, and I wasn't going to invert my autostarts for one character. This script now checks if a tarantula exists, like almanac.lic checks for an almanac, and exits if none is found.

While there, I added some DR* prefixes as well.

### CODE REVIEW

File: tarantula.lic [hunk 1]
```diff
@@ -7,21 +7,24 @@ custom_require.call(%w(common drinfomon))
...
-  include DRC
 
   def initialize
     settings = get_settings
     @tarantula = settings.tarantula
-    @startup_delay = settings.tarantula_startup_delay || 15
-    @tarantula_noun = settings.tarantula_noun.nil? ? 'biomechanical tarantula' : settings.tarantula_noun
+    @startup_delay = settings.tarantula_startup_delay
+    @tarantula_noun = settings.tarantula_noun
+    unless DRCI.exists?(@tarantula_noun)
```
Comment (rpherbig) [inline tarantula.lic:16]: `common-items` needs to be added to `custom_require` near the top

Comment (MahtraDR) [reply tarantula.lic:16]: Do you use an IDE that quickly points that out to you, or do you just the The Eye for that kind of thing? I'm using vim for these, and seriously considering going RubyMine, or VSCode if they'll help me spot this kind of thing easier.

Comment (asechrest) [reply tarantula.lic:16]: > Do you use an IDE that quickly points that out to you, or do you just the The Eye for that kind of thing? I'm using vim for these, and seriously considering going RubyMine, or VSCode if they'll help me spot this kind of thing easier.

I use VSCode and it doesn't help me with this, but maybe I don't have it set up right.

I think @rpherbig is just damn good at this. Hah.

Comment (rpherbig) [reply tarantula.lic:16]: I don't use an IDE, just GitHub's diff interface

**Pull request merged**